partiql-lib/partiql-sprout/src/main/kotlin/org/partiql/sprout/generator/target/kotlin/KotlinCommand.kt

```diff
@@ -40,14 +41,28 @@ class KotlinCommand : Callable<Int> {
     )
     lateinit var out: Path
 
+    @CommandLine.Option(
+        names = ["--poems"],
+        description = ["Poem templates to apply"],
+    )
+    var poems: List<String> = emptyList()
```

Comment:
That's not necessarily a recommendation, just showing you one way to use it. I used var + emptyList to have a default value thereby making the poem list optional. Without that you get

```kotlin.UninitializedPropertyAccessException: lateinit property poems has not been initialized
	at org.partiql.sprout.generator.target.kotlin.KotlinCommand.call(KotlinCommand.kt:63)
	at org.partiql.sprout.generator.target.kotlin.KotlinCommand.call(KotlinCommand.kt:18)
	at picocli.CommandLine.executeUserObject(CommandLine.java:2041)
	at picocli.CommandLine.access$1500(CommandLine.java:148)
	at picocli.CommandLine$RunLast.executeUserObjectOfLastSubcommandWithSameParent(CommandLine.java:2461)
	at picocli.CommandLine$RunLast.handle(CommandLine.java:2453)
	at picocli.CommandLine$RunLast.handle(CommandLine.java:2415)
	at picocli.CommandLine$AbstractParseResultHandler.execute(CommandLine.java:2273)
	at picocli.CommandLine$RunLast.execute(CommandLine.java:2417)
	at picocli.CommandLine.execute(CommandLine.java:2170)
	at org.partiql.sprout.SproutKt.main(Sprout.kt:24)
```

And if you try `defaultValue = ""` it will create a list with one element `listOf("")`. 

As is makes emptyList the default value while keeping `--poems` optional